tests/health_checks/NDv4/test-ib.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+##This script examines the output of ibstat, looking for any problems.
+
+#Catch error codes that may be thrown by the executable passed as the first
+#input, and if an error code is tripped throw the second input as a message
+catch_error() {
+	output=$($1)
+	err_code=$?
+	if [ $err_code -ne 0 ]; then
+		echo -e "\t $2 $err_code" >&2
+		exit $err_code;
+	fi
+}
+echo "check ibstat:"
+
+#Store output from calls passed to catch_error
+output=""
+
+ib=200;
+num_r=0
+num_s=0
+num_ps=0
+
+#Check ibstat Rates and make sure they are all 200.
+exec_ibstat="timeout 3m ibstat"
```

Comment:
this will fail if AN is enabled :(..

we can use ibstatus here (with the relevant postprocessing).

Reply:
Thanks, I have switched to ibstatus and modified the postprocessing appropriately.